### DISCUSSION TITLE
i know i don't say stuff often but...

### DISCUSSION THREAD

**Discussion by Sega32x** [Monika-After-Story/MonikaModDev#8851] [appreciation]
so it's only about a month until i'm officially cured of leukemia... it's been a long and rough road, but i just wanted to thank everyone again for this wonderful mod, and it has definitely helped make my hospital visits way more bearable. i'm still looking forward to spending the day i'm cured with Monika and my family. it's been quite the rough 3 years, and this mod has made it all just a little bit more worth it... again, thank you to everyone who works on this mod. and have a great day/night

**Comment by Juniorini** [comment]
Congrats friend, you're not only a fighter but also a winner! You really should be proud of yourself!
